tox.ini

```diff
@@ -399,7 +415,8 @@ passenv=TEST_*
 
 commands =
 # run only essential tests related to the tracing client
-    tracer: pytest {posargs} --ignore="tests/contrib" --ignore="tests/test_integration.py" --ignore="tests/commands" --ignore="tests/opentracer" --ignore="tests/unit" --ignore="tests/internal" tests
+    tracer: pytest {posargs} --ignore="tests/contrib" --ignore="tests/test_integration.py" --ignore="tests/commands" --ignore="tests/opentracer" --ignore="tests/unit" --ignore="tests/internal" --ignore="tests/profile" tests
```

Comment:
we should fix this, move stuff into `tests/tracer/` or something.

(just an aside)